bark_ml/library_wrappers/lib_tf2rl/agents/gail_agent.py

```diff
@@ -38,37 +38,39 @@ def _get_generator(self):
         """Returns instantiated policy -
         parameters from ./examples/example_params/gail_params.json
         """
+        local_params = self._params["ML"]["BehaviorGAILAgent"]
```

Comment:
Suggestion of Tobias.

Reply:
Can you please make it even one deeper. So down to `self._params["ML"]["BehaviorGAILAgent"]["Generator"]` and rename to `generator_params`?